src/util/BitField.js

```diff
@@ -41,6 +41,7 @@ class BitField {
    * @returns {boolean}
    */
   has(bit) {
+    if (typeof bit === 'undefined') return false;
     if (Array.isArray(bit)) return bit.every(p => this.has(p));
     bit = this.constructor.resolve(bit);
```

Comment:
```suggestion
    if (Array.isArray(bit)) return bit.every(p => this.has(p));
    bit = this.constructor.resolve(bit);
    if (bit === this.constructor.defaultBit) return false;
```

Reply:
I hadn't thought of it that way, but it seems to be a more generic fix 👍